content/applications/general/email_communication/email_servers.rst

```diff
@@ -1,222 +1,209 @@
-Send and Receive Emails in Odoo with an Email Server
+Send and receive emails in Odoo with an email server
 ====================================================
 
-If you are a user of Odoo Online or Odoo.sh...
-==============================================
+Odoo Online or Odoo.sh users
+============================
 
-You have nothing to do! **Odoo sets up its own mail servers for your database.**
-Outgoing and incoming emails work out-of-the-box!
-
-Unless you plan to send large batches of mass mailing that could require the 
-use of an external mail server, simply enjoy your new Odoo database.
+Nothing needs to be done! **Odoo sets up its own mail servers for the database.** Outgoing and
+incoming emails work out-of-the-box!
 
+Unless an external mail server is required to send large batches of mass mailing, simply enjoy the
+new Odoo database.
 
 Scope of this documentation
 ===========================
 
-This document is **mainly dedicated to Odoo on-premise users** who don't
-benefit from an out-of-the-box solution to send and receive emails in Odoo,
-unlike `Odoo Online <https://www.odoo.com/trial>`_ & `Odoo.sh <https://www.odoo.sh>`_.
-
+This document is **mainly dedicated to Odoo on-premise users** who don't benefit from an
+out-of-the-box solution to send and receive emails in Odoo, unlike `Odoo Online <https://www.odoo.
+com/trial>`_ & `Odoo.sh <https://www.odoo.sh>`_.
 
 .. warning::
-    If no one in your company is used to manage email servers, we strongly recommend that
-    you opt for those Odoo hosting solutions. Their email system
-    works instantly and is monitored by professionals.
-    Nevertheless you can still use your own email servers if you want
-    to manage your email server's reputation yourself.
-
-You will find here some useful
-information on how to integrate your own email solution with Odoo.
-
-.. note:: 
-   Office 365 email servers don't easily allow to send external emails from hosts like Odoo. Refer
-   to `Microsoft's documentation <https://support.office.com/en-us/article/How-to-set-up-a-multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-c805d654b4c4>`_ 
-   to make it work.
-
-How to manage outbound messages
-===============================
-As a system admin, go to :menuselection:`Settings --> General Settings`
-and check *External Email Servers*. Then, click *Outgoing Mail Servers* to create one and reference 
-the SMTP data of your email server. Once all the information has been filled out, click on 
-*Test Connection*.
+   If no one in the company is used to managing email servers, Odoo Online and Odoo.sh are strongly
+   recommended. Those Odoo hosting types' email systems work instantly and are monitored by
+   professionals. Nevertheless, a company can use their own email server if they want to manage the
+   email server's reputation themselves.
+
+The following sections contain information on how to integrate an external email server with Odoo.
```

Comment:
Would move this sentence above the `.. warning::` tag since it's a continuation of that first paragraph.
Would also add "below" to indicate proceeding information as user scrolls down.
```suggestion
The following sections below contain information on how to integrate an external email server with
Odoo.
```